UT4MasterServer.Models/Database/Account.cs

```diff
@@ -89,7 +89,7 @@ public float Level
 			}
 
 			// note: req to next level, so element 0 is XP required for level 1
-			ulong xp = 0;
+			ulong xp;
```

Comment:
I think it can be nice to have an explicit initialization value which makes the code more clear. But it's fine if IDE suggests to remove it.

Please edit editorconfig so that compiler won't issue a warning for this.

Reply:
Docs: https://www.jetbrains.com/help/resharper/2024.1/RedundantDefaultMemberInitializer.html

But OK, I will edit editorconfig to disable this